src/BBT.StructureTools.Tests/TestTools/IoC/IIocContainer.cs

```diff
@@ -19,6 +19,7 @@ internal interface IIocContainer
         /// <typeparam name="TAbstraction">Abstraction (interface) type.</typeparam>
         /// <typeparam name="TImplementation">Implementation of <typeparamref name="TAbstraction"/>.</typeparam>
         void RegisterSingleton<TAbstraction, TImplementation>()
+            where TAbstraction : class
```

Comment:
Because Castle Windsor's `Component.For<TAbstraction>` requires a reference type. The `class` constraint in C# doesn't require a class but a reference type.